### PR TITLE
Don't show children without active placements in calendar view

### PR DESCRIPTION
**Pull request by zch**

### CODE REVIEW

File: frontend/src/lib-customizations/espoo/citizen/assets/i18n/en.tsx [hunk 1]
```diff
@@ -260,6 +260,8 @@ const en: Translations = {
     reservation: 'Registered attendance',
     realized: 'Realized',
     reservationsAndRealized: 'Period of attendance',
+    noActivePlacements:
+      'Your child does not have an active placement in the early childhood education or pre-school on this day.',
```
Comment (jaahon) [inline frontend/src/lib-customizations/espoo/citizen/assets/i18n/en.tsx:264]: I'm not sure about this copy. The concept of "active placements" doesn't seem like something that would be obvious to our users. Would something like "Your children are not in early childhood education or pre-school on this day" be better? What do you think?

Comment (zch) [reply frontend/src/lib-customizations/espoo/citizen/assets/i18n/en.tsx:264]: I like your suggestion, although "Your children are not in early childhood education or pre-school on this day" is also correct for sick days... But it still might be better. I'll run it by Anna 👍

Comment (zch) [reply frontend/src/lib-customizations/espoo/citizen/assets/i18n/en.tsx]: Simplified the copy in all languages 👍

**Review by jaahon — APPROVED** [review]

File: frontend/src/lib-customizations/espoo/citizen/assets/i18n/en.tsx (hunk 1 above)
Comment (jaahon) [reply frontend/src/lib-customizations/espoo/citizen/assets/i18n/en.tsx]: It's very good now!

**Pull request merged**